compose/config/config_schema_v3.7.json

```diff
@@ -151,6 +151,8 @@
 
         "external_links": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
         "extra_hosts": {"$ref": "#/definitions/list_or_dict"},
+        "device_requests": {"$ref": "#/definitions/device_requests"},
```

Comment:
Within schema v3, runtime constraints have been moved into `deployment.resources`. Sounds a better place for such device allocation request

Reply:
I can move but seems to me like it's better to wait until you figure out how to reconcile schema definitions between docker and compose before making any other change ...